db/toblock.c

```diff
@@ -5624,6 +5627,7 @@ static int toblock_main_int(struct javasp_trans_state *javasp_trans_handle,
                     assert(outrc || iq->sc_running == 0);
                     iq->sc_logical_tran = NULL;
                 } else {
+                    // TODO: Is parent_trans still valid after this?  Does it matter?
```

Comment:
Yup- parent_trans will be invalid after this point- you can guarantee that we will segfault if you set it to NULL here